.do/deploy.template.yaml

```diff
@@ -0,0 +1,14 @@
+spec:
+  name: fasten-onprem
+  services:
+    - dockerfile_path: Dockerfile
+      name: app
+      git:
+        branch: add-deploy-to-digital-ocean-button
+        repo_clone_url: https://github.com/cfu288/fasten-onprem.git
```

Comment:
Needs to be updated to `https://github.com/fastenhealth/fasten-onprem` before merge